<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trading</string>
  </property>
  <widget class="QLineEdit" name="account_label0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_label1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="acount1_label">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>58</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LogonID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="account2_label">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>LogonID:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accountLogin">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="torder">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="label0_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>185</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label0_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>185</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label0_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>185</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label0_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>185</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label0_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>185</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label1_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>215</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label1_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>215</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label1_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>215</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label1_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>215</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label1_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>215</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label2_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>245</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label2_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>245</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label2_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>245</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label2_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>245</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label2_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>245</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label3_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>275</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label3_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>275</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label3_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>275</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label3_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>275</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label3_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>275</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label4_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>305</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label4_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>305</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label4_0">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>305</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label4_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>305</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="label4_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>305</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LogonID_label0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogonID_label1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>LogonID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>162</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Wind Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Trade Direction</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>477</x>
     <y>162</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>162</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>189</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Order 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Order 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Order 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Order 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Order 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="logon_msg">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Logon Msg:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="logon_msg_browser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logon_msg_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Msg:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="order_msg_browser_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
